### DISCUSSION TITLE
Unable to install on VSCode

### DISCUSSION THREAD

**Discussion by AnOliveBranch** [community/community#7518] [Copilot Conversations] [answered]
So I'm trying to get copilot working, but had some problems. First, it wouldn't show up in the extension market at all. So I downloaded the VSIX and installed from there. Got authenticated, now it's saying I need VS Code 1.57+. My current VS Code version is 1.62.
![image](https://user-images.githubusercontent.com/45128900/140984001-35cf1643-f3c9-4bf3-8823-d264a65f1a1d.png)
![image](https://user-images.githubusercontent.com/45128900/140984024-9a1b673e-d16a-4fae-96f7-94f7c9c1b90f.png)
And pacman doesn't have an update available for VS Code either.

Edit: Also I'm using v1.7.3733
![image](https://user-images.githubusercontent.com/45128900/140986637-9de09543-561b-42d1-859b-577c9ded9bd8.png)

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Please check this comment - https://github.com/github/feedback/discussions/6629#discussioncomment-1524627
